sysctl.d/50-coredump.conf.in

```diff
@@ -13,7 +13,7 @@
 # the core dump.
 #
 # See systemd-coredump(8) and core(5).
-kernel.core_pattern=|{{LIBEXECDIR}}/systemd-coredump %P %u %g %s %t %c %h
+kernel.core_pattern=|{{LIBEXECDIR}}/systemd-coredump %P %u %g %s %t %c %h %F
```

Comment:
what do old kernels actually do if we specify %F? generate an empty parameter?

Reply:
they ignore it

Reply:
In what sense? Do they generate an empty argument, or no argument at all?

Reply:
empty string arg